constants.py

```diff
@@ -6,4 +6,10 @@
 TELEGRAM_API_TOKEN = env('TELEGRAM_API_TOKEN')
 GITHUB_SECRET = env('GITHUB_SECRET')
 
-CONVERSATIONS = env.list('CONVERSATIONS', [131845033, -1001342903759], subcast=int)
\ No newline at end of file
+CONVERSATIONS = env.list('CONVERSATIONS', [519013710], subcast=int)
+
+HEALTHCHECK_SERVICES = {'LH website':'https://lh.imesec.ime.usp.br',
+                        'LH API':'https://api.lh.imesec.ime.usp.br/hacks'}
+
+HEALTHCHECK_SLEEP_AMOUNT = 2 * 60
+HEALTHCHECK_ANOMALY_THRESHOLD = 5
```

Comment:
Since we allowed the sleep amount to be two minutes, we can lower the anomaly threshold to `3` again, since there'll be less fluctuation in a longer time between requests.